std/http/cookie_test.ts

```diff
@@ -31,6 +31,28 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "Cookie Validation",
+  fn(): void {
+    const res: Response = {};
+    let error = false;
+    res.headers = new Headers();
+    try {
+      setCookie(res, {
+        name: "Name",
+        value: "中文名字",
+        httpOnly: true,
+        secure: true,
+        maxAge: 0,
+      });
+    } catch (e) {
+      error = true;
+    }
+    assert(error);
```

Comment:
[`assertThrows()`](https://github.com/denoland/deno/blob/master/std/testing/asserts.ts#L425) from `std/testing/asserts` is a powerful tool for these kind of checks. 😉

Reply:
Hi @schwarzkopfb 
If you like all testing file used in deno. They use this approach.

Reply:
@getspooky `assertThrows()` is the way to go.

Reply:
@nayeemrmn when i run deno test std/http/cookie_test.ts i got 
```sh
TS2345 [ERROR]: Argument of type '{ depth: number; sorted: boolean; trailingComma: boolean; compact: boolean; iterableLimit: number; }' is not assignable to parameter of type 'InspectOptions'.
  Object literal may only specify known properties, and 'sorted' does not exist in type 'InspectOptions'.
      sorted: true,
      ~~~~~~~~~~~~
    at file:///home/getspooky/Documents/OpenSourceProject/deno/std/testing/asserts.ts:26:7
```

Reply:
@getspooky looks like you're running on old version of Deno

Reply:
@bartlomieju thanks , I forgot to update deno version